tests/Integration/Behaviour/behat.yml

```diff
@@ -472,6 +394,76 @@ default:
             contexts:
                - Tests\Integration\Behaviour\Features\Context\CommonFeatureContext
                - Tests\Integration\Behaviour\Features\Context\Domain\CountryFeatureContext
+        product:
```

Comment:
Ok I get it so moved this one closer to the update_product suite That makes sense
Final step will be to do the same cleaning for the unified combination command and then we can clear the update_pproduct suite and only keep the product one